app/src/main/java/uw/hcrlab/kubi/App.java

```diff
@@ -68,7 +68,7 @@ public static void initImageLoader(Context context) {
 
     public static void FbConnect() {
         fb.child("devices").child(deviceID).child("connected").setValue(true);
-        fb.child("devices").child(deviceID).child("name").setValue(deviceName);
+        fb.child("devices").child(deviceID).child("name").setValue("Alex's Nexus 7");
```

Comment:
Yeah. I need to come up with a better solution, but this was a temporary fix to make the Chrome extension differentiate between our devices. You should set it to your name in your branch. I'll work on a better fix